local-tests/test_update.ipynb

```diff
@@ -28,8 +28,8 @@
     "newbin = join(workdir,'aleph-node-new')\n",
     "# Path to the post-update compiled runtime:\n",
     "runtime = join(workdir,'aleph_runtime.compact.wasm')\n",
-    "# Path to the send-runtime binary (which lives in aleph-node/local-tests/send-runtime):\n",
-    "send_runtime = 'send-runtime/target/release/send_runtime'"
+    "# Path to the cliain:\n",
```

Comment:
Unneeded definite article, either remove "the" or add "binary" after "cliain".

Reply:
eagle eye :D